src/components/Tables/Filters.tsx

```diff
@@ -44,3 +51,9 @@ export const defaultToString = (value: unknown): string => {
 
     return JSON.stringify(value);
 };
+
+export interface CoreFilter<State = any> {
```

Comment:
we use this in TableFilterAndExtraColumnsBar.tsx

Reply:
CoreFilter is only used in the TableFilterAndExtraColumnsBar to extend the Filter type

Reply:
have deleted as per the below suggestion